gemini/sample-apps/fashion-trends-prediction/app/pages/2_     ● Fashion Bot.py

```diff
@@ -0,0 +1,255 @@
+import base64
+import io
+import json
+import time
+
+from config import config
+import streamlit as st
+from utilities import *
+from utils_standalone_image_gen import *
+import vertexai
+from vertexai.generative_models import GenerativeModel, Part
+import vertexai.preview.generative_models as generative_models
```

Comment:
Not sure if you need this extra import. It could be easier to import the `HarmCategory` and `HarmBlockThreshold` classes directly.

Reply:
Done in all the files